README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
> then it would not pass. but the change would not change code behavior

True, but it hasn't made a correct change though. It's renamed a function that wasn't a native function.